extras/exporters/gltf-exporter.js

```diff
@@ -365,7 +416,45 @@ class GltfExporter {
         }
     }
 
-    buildJson(resources) {
+    convertTextures(textures, json, options) {
+
+        const textureOptions = {
+            maxTextureSize: options.maxTextureSize
+        };
+
+        for (let i = 0; i < textures.length; i++) {
+
+            // for now store all textures as png
+            // TODO: consider jpg if the alpha channel is not used
```

Comment:
Wouldn't you just use the pixel format of the texture to decide?

Reply:
yes that's the plan.